specification/2.0/schema/accessor.schema.json

```diff
@@ -39,10 +65,33 @@
             "gltf_detailedDescription": "The number of attributes referenced by this accessor, not to be confused with the number of bytes or number of components."
         },
         "type": {
-            "type": "string",
             "description": "Specifies if the attribute is a scalar, vector, or matrix.",
-            "enum": [ "SCALAR", "VEC2", "VEC3", "VEC4", "MAT2", "MAT3", "MAT4" ],
-            "gltf_detailedDescription": "Specifies if the attribute is a scalar, vector, or matrix, and the number of elements in the vector or matrix."
+            "anyOf": [
+                {
+                    "enum": [ "SCALAR" ]
+                },
+                {
+                    "enum": [ "VEC2" ]
+                },
+                {
+                    "enum": [ "VEC3" ]
+                },
+                {
+                    "enum": [ "VEC4" ]
+                },
+                {
+                    "enum": [ "MAT2" ]
+                },
+                {
+                    "enum": [ "MAT3" ]
+                },
+                {
+                    "enum": [ "MAT4" ]
+                },
+                {
+                    "type": "string"
```

Comment:
Same comment here with `string`.  I'm not a JSON Schema expert, but I tried actually loading this branch into VSCode's JSON validator, and having this `type` at the end of the enum list allows arbitrary strings (or arbitrary ints in the comment above) to validate as correct.  Removing this `type` means it will only validate with an exact match to one of the named enum values.